packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -348,14 +348,25 @@ class CliServiceProvider
           { getParameter: 1, featureCompatibilityVersion: 1 },
           this.baseCmdOptions
         ).catch(() => {}),
+        this.runCommandWithCheck(
+          'admin',
+          {
+            count: 'atlascli',
+            query: { managedClusterType: 'atlasCliLocalDevCluster' },
+          },
+          this.baseCmdOptions
+        ).catch(() => ({ n: 0 })),
       ]);
 
+    const isLocalAtlasCli = Number(atlascliInfo.n) > 0;
```

Comment:
```suggestion
        this.countDocuments(
          'admin',
          'atlascli',
          { managedClusterType: 'atlasCliLocalDevCluster' }
        ).catch(() => 0),
      ]);

    const isLocalAtlasCli = !!atlascliInfo;
```

Given that there has been a lot of back-and-forth around the `count` command’s level of stability and the `count()` driver helper I think it’s best to just stick with a non-deprecated driver helper here

Reply:
Thanks for suggesting this. I updated it in [4a2ecd0](https://github.com/mongodb-js/mongosh/pull/1629/commits/4a2ecd0db82e12794a1e4f3c9f3377d860875309)